src/shapes/text.class.js

```diff
@@ -950,8 +954,8 @@
 
       var line = this._textLines[lineIndex], maxHeight = 0;
       for (var i = 0, len = line.length; i < len; i++) {
-        maxHeight = Math.max(this.getHeightOfChar(lineIndex, i), maxHeight);
-        // TODO: Take 'deltaY' into account to avoid overlaps (may require 'allowOverlaps' property)
+        var box = this._getGraphemeBox(null, lineIndex, i);
+        maxHeight = Math.max(maxHeight, box.height);
```

Comment:
i would not change getHeightOfLine, i know devs will be able to set values that overlap, but i do not want to change basic behaviours for a new feature that has a marginal impact on overall library usage.